pillar.example

```diff
@@ -1,19 +1,19 @@
 postgres:
-  # Set True to configure upstream postgresql.org repository for YUM or APT
+  # UPSTREAM REPO
+  # Set True to configure upstream postgresql.org repository for YUM/APT/ZYPP
   use_upstream_repo: False
-  # Version to install from upstream repository
-  version: '9.3'
-
-  # These are Debian/Ubuntu specific package names
-  pkg: 'postgresql-9.3'
-  pkg_client: 'postgresql-client-9.3'
-
-  # Additional packages to install with PostgreSQL server,
-  # this should be in a list format
-  pkgs_extra:
-    - postgresql-contrib
-    - postgresql-plpython
-
+  # Version to install from upstream repository (if upstream_repo: True)
+  version: '9.6'
+
+  # PACKAGE
+  # These pillars are typically never required.
+  # pkg: 'postgresql'
+  # pkg_client: 'postgresql-client'
+  # service: postgresql
+  #pkgs_extra:
```

Comment:
This `pkg_extra` parameter still concerns me.

I thought we reached agreement that `pillar.example` file could not be used as valid source for any possible platform which the formula supports. Hard-coding some package names is bad thing too.
But now this breaks the test kitchen run.
We have two options:
* Uncomment `pkgs_extra` and add `postgresql-contrib` there mentioned this could be used only with `use_upstream_repo: False`.
* ...or replace `uuid-ossp` extension with `plpgsql`, that comes with PG server. The schema name also should be changed to some `custom_schema` for consistency.

Reply:
I have reinstated `pkgs_extra` as per original pillar.example.   

Yeah, I'm unconvinced about pillar.example. The README and pillar.example are both delivered artifacts, responsible for onboarding new formula users (perhaps in a 1 week agile sprint), and guiding adoption for happy and unhappy paths. test kitchen is a platform too.   **On the other hand, I agree the pillar.example must have limited scope**, which aligns with your stated view.